### PR TITLE
Post auction

### PR DESCRIPTION
**Pull request by mikan373**
## What
POST request to enter bid price

## How
1. List an item from frontend
2. Check its uuid by getting items
```
curl -X GET http://localhost:9000/items
```
3. Use that uuid for POST request to post bid price
```
curl -X POST http://localhost:9000/auction/<uuid>
-d "items_id=<uuid>" -d "bid_price=5000"
```

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -255,4 +256,41 @@ def update_status(id):
...
+    
+    try:
+        # generate UUID
+        generated_id = str(uuid.uuid4())
+    
+        c.execute("SELECT name FROM items WHERE id = (?)", (item_id,))
+        data = c.fetchall()
+        
+        (name, ) = data[0]
+        print(name)
```
Comment (mfunyu) [inline python/main.py:274]:
```suggestion
```

File: python/main.py [hunk 2]
```diff
@@ -255,4 +256,41 @@ def update_status(id):
...
+    except sqlite3.IntegrityError as err:
+        if "UNIQUE constraint" in str(err):
+            raise HTTPException(status_code=400, detail=f"This user already has a bid for this item")
+            logger.error(f"Unhandled error occurred")
+            raise err
+    finally:
+        conn.close()
+
+    return {data[0]}
+    #return {"message": f"item received: {name}"}
```
Comment (mfunyu) [inline python/main.py]:
```suggestion
```

**Review by mfunyu — APPROVED** [review]
You might want to delete these lines, but otherwise its all good!

File: python/main.py (hunk 1 above)
Comment (mfunyu) [inline python/main.py]:
```suggestion
```

**Review by dominic-soh — APPROVED** [review]

**Pull request merged**